README.md

```diff
@@ -26,6 +26,9 @@ of the protocol in Rust, as well as exhaustive testing. The layout of the reposi
 - `timeboost-crypto`: Contains the threshold encryption scheme used by timeboost.
 - `timeboost-utils`: Contains some utility functions.
 
+## MSRV (minimum supported Rust version)
+The MSRV of this repository is 1.85.
```

Comment:
Maybe `Cargo.toml` can then be extended

```diff
diff --git i/Cargo.toml w/Cargo.toml
index 45dcde9..818f464 100644
--- i/Cargo.toml
+++ w/Cargo.toml
@@ -19,6 +19,7 @@ resolver = "2"
 [workspace.package]
 version = "0.1.0"
 edition = "2021"
+rust-version = "1.85"
 description = "Low-latency decentralized sequencer with optional encrypted mempool"

 [profile.test]
```

and the individual packages link to this version. We could also do this in a follow-up PR.

Reply:
TIL! Thanks